.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps = true
```

Comment:
Will this be removed when the new version of Source is available?

Reply:
yes, I've removed it!